### PR TITLE
Update metadata.yaml

### PR DESCRIPTION
**Pull request by soumitagh**
update author information.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by agitter — COMMENTED** [review]
Thanks for the code of conduct approval.

If you still have no conflicts of interest to note, can you please change the line
`lastapproved: !!str 2020-04-28`
to
`lastapproved: !!str 2020-11-09`

File: content/metadata.yaml [hunk 1]
```diff
@@ -500,7 +500,7 @@ authors:
     contributions:
       - Writing - Original Draft
     code of conduct:
-      confirmed: No
+      confirmed: yes
     affiliations:
       - Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     coi:
```
Comment (soumitagh) [inline content/metadata.yaml:506]: 2020-9-11

Comment (soumitagh) [inline content/metadata.yaml:506]: coi
lastapproved: !!str 2020-11-9

Comment (ajlee21) [reply content/metadata.yaml:506]: So I think you need to go in and edit the `metadata.yml` file in the same way you updated the `confirmed: yes` text and commit and push those changes to your `patch-8` branch. Does that make sense?

**Review by rando2 — APPROVED** [review]
I made the edits suggested by @agitter and @ajlee21 -- I know the branches can get confusing, so FYI the easiest way to do it is to go to the "Files changed" tab of the PR and then click on the three dots in the upper right hand corner of where it renders the text from the file! I hope that helps for the future. Thanks for making these updates!

**Pull request merged**